test-clients/src/main/java/com/hedera/services/bdd/spec/utilops/UtilVerbs.java

```diff
@@ -414,35 +415,52 @@ public static HapiSpecOperation ensureDissociated(String account, List<String> t
 	}
 
 	public static HapiSpecOperation makeFree(HederaFunctionality function) {
+		return reduceFeeFor(function, 0L, 0L, 0L);
+	}
+
+	public static HapiSpecOperation reduceFeeFor(HederaFunctionality function,
+			long tinyBarMaxNodeFee, long tinyBarMaxNetworkFee, long tinyBarMaxServiceFee) {
 		return withOpContext((spec, opLog) -> {
+			if (!spec.setup().defaultNode().equals(asAccount("0.0.3"))) {
+				opLog.info("Sleeping to wait for fee reduction...");
+				Thread.sleep(20000);
+				return;
+			}
+			Thread.sleep(10000);
```

Comment:
What's the purpose to wait 10s here?

Reply:
At first, I added only the sleep on line 426 for other clients to wait for the fee schedules update. Then as described in the following commit https://github.com/hashgraph/hedera-services/pull/1144/commits/d4a0d63ee802eb1656701c9596428705c1797f04, because `HapiApiSpec.tryReinitializingFees()` runs before any other ops, so on other clients, `HapiApiSpec.tryReinitializingFees()` will run before the sleep on line 426 kicks in. The client targeting node 0.0.3 has to wait so not to disturb (fail) the `HapiApiSpec.tryReinitializingFees()` on other clients. I'll add comment to explain in the test code too.

Reply:
I see. Yeah, I think a comment will make it clear. I was thinking the client for 0.0.3 to run first, but forgot other clients may run earlier than node 0.0.3's client.